src/npip.js

```diff
@@ -63,11 +63,11 @@ const installPip = async (pkg) => {
     // run get-pip.py
     await spawnPython([tmpFile, "--user", "--quiet"], opt);
     console.log("Successfully completed pip check.");
-    fs.unlinkSync(tmpFile);
     return;
   } catch (error) {
-    fs.unlinkSync(tmpFile);
-    console.error("Failed to install pip locally.\n" + JSON.stringify(error));
+    console.error("Failed to install pip locally.\n" + JSON.stringify(error, null, 2));
```

Comment:
should this be `throw` instead? since if we got to this point, things are pretty doomed

Reply:
Opted for removing the catch block